sql/hive/src/test/scala/org/apache/spark/sql/hive/test/TestHive.scala

```diff
@@ -601,9 +602,12 @@ private[hive] class TestHiveQueryExecution(
     }
 
     // Make sure any test tables referenced are loaded.
-    val referencedTables =
-      describedTables ++
-        logical.collect { case UnresolvedRelation(ident, _, _) => ident.asTableIdentifier }
+    val referencedTables = describedTables ++ logical.collect {
+      case UnresolvedRelation(ident, _, _) =>
+        if (!ident.isEmpty && ident.head.equalsIgnoreCase(CatalogManager.SESSION_CATALOG_NAME)) {
```

Comment:
`ident.length > 1`, as table name must have at least one part.

Reply:
This is checked inside of `asTableIdentifier`. Here, we need at least one element.

Reply:
Actually, it's different. For `SELECT * FROM spark_catalog`, `spark_catalog` should be a table name, not failing.